assets/src/components/searchForm.tsx

```diff
@@ -25,6 +31,9 @@ const SearchForm = () => {
   const clearTextInput = (event: React.FormEvent<EventTarget>): void => {
     event.preventDefault()
     dispatch(setSearchText(""))
+    if (onClear) {
+      onClear()
+    }
```

Comment:
**suggestion:** even if this and the `onSubmit` callback functionality are hit by test cases in other components, it would be good to add test cases to `searchForm.test.tsx` directly.